src/legacy/router.js

```diff
@@ -366,7 +371,9 @@ async function handleValidation(ctx) {
       const hv = await eventValidator.handleValidation(parsedEvent);
       sendViolationMetrics(hv.validationErrors, hv.dropEvent, metaTags);
       if (hv.dropEvent) {
-        const errMessage = `Error occurred while validating because : ${hv.violationType}`;
+        const errMessage = `Error occurred while validating due to: ${
+          hv.violationType
+        }\n Other errors: ${JSON.stringify(getValidationErrMsg(hv.validationErrors))}`;
```

Comment:
new line required? in loki it gets difficult to search?

Reply:
main purpose is to show this as sampleResponse in events tab